src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck has stopped working.");
```

Comment:
Similarly, the stop message should be more informative. Consider changing 'Truck has stopped working.' to 'Truck has finished its work.' to provide clearer information.